frontend/src/components/table/Table.js

```diff
@@ -174,47 +193,67 @@ export default class Table extends PureComponent {
     }
 
     switch (e.key) {
-      case 'ArrowDown': {
+      case ARROW_DOWN_KEY: {
         e.preventDefault();
 
-        const { currentId, array } = this.getCurrentRowId();
+        const { currentId, rowsPool } = this.getCurrentRowId(ARROW_DOWN_KEY);
 
-        if (currentId >= array.length - 1) {
-          return;
-        }
+        if (currentId >= rowsPool.length - 1) return;
 
         if (!selectRange) {
           handleSelect(
-            array[currentId + 1],
+            rowsPool[currentId + 1],
             false,
             idFocused,
             showSelectedIncludedView &&
-              showSelectedIncludedView([array[currentId + 1]])
+              showSelectedIncludedView([rowsPool[currentId + 1]])
           );
+          this.clearMultiSelectionStartIdx();
         } else {
-          handleSelect(array[currentId + 1], false, idFocused);
+          this.updateMultiSelectionStartIdx(currentId);
+
+          const downShiftSel = rowsPool.slice(
+            this.multiSelectionStartIdx > 0 ? this.multiSelectionStartIdx : 0,
+            currentId + 2 // +2 because we want to slice up to the next row and include it
```

Comment:
why also next row?
what's the reason behind?

Reply:
If you put 1 it will remain on the exact same row where it was all the time